File: src/TE.java
public class TE{
	String dest,
			nexthop;
	Port port;

	public TE(String net_dest,String nexthop, Port port){
		this.dest = net_dest;
		this.nexthop = nexthop;
		this.port = port;
	}

	public String toString(){
		return dest+","+nexthop+","+port;
	}
}

File: src/te.cpp
#include <string>
#include "te.hpp"
using namespace std;

string te::getNextHopIP(){
    return this->nextHop;
}

string te::toString(){
    stringstream ss;
    ss << this->getNetworkDest() << " | " << this->getNextHopIP() << " : " << this->getPort()->getPortNumber();
    return ss.str();
}
